### ISSUE TITLE
OOM on owt training

### ISSUE THREAD

**Issue by david3684** [s-sahoo/duo#17]
Hello, thank you for the great work!

I'm using 8xH100 GPUs and tried to run `train_owt_duo.sh` to train on openwebtext dataset, as the same setting as `loader.batch_size=32`, but i get gpu OOM on first training step. When i reduce to `loader.batch_size=16`, it runs with consuming about ~51GB VRAM per GPU.

My question is: are the 8 H100 GPUs that you reported on the paper are 96GB GPUs? 
Then it makes sense since I am using 80GB H100s; if it's not, i don't get the reason why under the same setting i'm running out of the memory. 

Thanks!

**Comment by s-sahoo** [comment]
That sounds about right! We used gradient accumulation for the OWT experiments.
